### DISCUSSION TITLE
Email wallets and third-party cookies

### DISCUSSION THREAD

**Discussion by jplsaez** [WalletConnect/walletconnect-monorepo#4546] [Developer Support]
Hello.

I realized that after adding "emailConnector" to my wagmi config to enable email wallets feature, lots of requests are continously sent from the user browser, most of them to "secure.walletconnect.com" and "auth.magic.link".

Chrome browser complaints with lots of warnings about "Third-party cookie will be blocked. Learn more in the Issues tab."

In the issues tab the following message is shown:

"Reading cookie in cross-site context will be blocked in future Chrome versions
Cookies with the SameSite=None; Secure and not Partitioned attributes that operate in cross-site contexts are third-party cookies. In future Chrome versions, reading third-party cookies will be blocked. This behavior protects user data from cross-site tracking.
Please refer to the article linked to learn more about preparing your site to avoid potential breakage."

It can be easyly reproduced in the following URL:

https://lab.web3modal.com/library/ethers-email/

Two points:

I guess third party cookies should be avoided.
Is there any way to prevent those requests to keep flowing forever?

Best regards.

Juan Pedro López

**Comment by Yuripetusko** [comment]
Getting this also. Followed the guide for Appkit. Using wagmi coookie storage in nextjs. Testing on localhost, I get thousands of warnings that just keep going forever

`Third-party cookie will be blocked in future Chrome versions as part of Privacy Sandbox.`

**Comment by ms-safiq** [comment]
Yeah i too get Third-party cookie will be blocked in future Chrome versions as part of Privacy Sandbox. like 10000 times in the console. Found that createWeb3Modal method is causing this.

**Comment by keplerbb01** [comment]
Has anyone found a solution?
